test/JDBC/upgrade/13_6/preparation/BABEL-2769-vu-prepare.sql

```diff
@@ -0,0 +1,33 @@
+CREATE DATABASE db_babel_2769;
```

Comment:
no need to use a separate database instead create objects with filename_xx as name, for example BABEL-2769-vu-prepare and so on

Reply:
creating all the tables in master db instead

Reply:
using master db instead